src/mscorlib/src/System/AppDomainUnloadedException.cs

```diff
@@ -17,27 +17,11 @@ namespace System {
 
     [System.Runtime.InteropServices.ComVisible(true)]
     [Serializable]
-    public class AppDomainUnloadedException : SystemException {
+    internal class AppDomainUnloadedException : SystemException {
         public AppDomainUnloadedException() 
             : base(Environment.GetResourceString("Arg_AppDomainUnloadedException")) {
             SetErrorCode(__HResults.COR_E_APPDOMAINUNLOADED);
         }
-    
-        public AppDomainUnloadedException(String message) 
-            : base(message) {
-            SetErrorCode(__HResults.COR_E_APPDOMAINUNLOADED);
-        }
-    
-        public AppDomainUnloadedException(String message, Exception innerException) 
-            : base(message, innerException) {
-            SetErrorCode(__HResults.COR_E_APPDOMAINUNLOADED);
-        }
-
-        //
-        //This constructor is required for serialization.
-        //
-        protected AppDomainUnloadedException(SerializationInfo info, StreamingContext context) : base(info, context) {
```

Comment:
I assume that an instance of this exception will never be thrown to user code?  If it could be, removing this ctor will break code if that code tries to serialize and deserialize the exception it caught.